lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3683,3 +3687,40 @@ def test_get_template_and_context(self):
             assert template == 'public_video_share_embed.html'
             assert context['fragment'] == fragment
             assert context['course'] == self.course
+
+class TestCoursewareMFESearchAPI(SharedModuleStoreTestCase):
+    """
+    Testing the Course Search API.
```

Comment:
needs new comment

Reply:
Replaced with `Tests the endpoint to fetch the Courseware Search waffle flag enabled status.`